### DISCUSSION TITLE
A deal is in State:StorageDealStaged, how to solve it?

### DISCUSSION THREAD

**Discussion by askender** [filecoin-project/lotus#5461] [Storage Provider] [answered]
lotus-miner storage-deals list |grep StorageDealStaged
...hsiaa5dy  1575112  StorageDealStaged                   f3**  32GiB     0 FIL                     1**

It maybe happens when `lotus-miner storage-deals import-data` is interrupted. and I want to cancel the deal( the start-epoch is 
 less then 24 hours now).

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment, marked as answer]
https://filecoinproject.slack.com/archives/C01KCAAURAN/p1611780886092300?thread_ts=1611735868.065100&cid=C01KCAAURAN

this might help you.
